Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx

```diff
@@ -532,12 +530,29 @@ void HeatExchanger::createManifold(Int_t disk)
   TGeoRotation* rcoverBodyStep = new TGeoRotation(Form("rcoverBodyStep4MF%d", disk), 45, 0, 0);
   rcoverBodyStep->RegisterYourself();
 
-  TGeoCompositeShape* coverBodyStep4rotated = new TGeoCompositeShape(Form("coverBodyStep4rotatedMF%d", disk), Form("dummy + coverBodyStep4MF%d:rcoverBodyStep4MF%d", disk, disk));
+  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+  //Add 01/07/2020
+  ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
do you need these date + // annotations? Probably a detailed explanation (in the git commit message) would be better.

Reply:
I removed these uneeded lines